Create wrangler sourcemaps guide for Cloudflare (#14089)
getsentry/sentry-docs: platform-includes/sourcemaps/overview/javascript.cloudflare.mdx

## Uploading Source Maps

The easiest way to configure uploading source maps is by using the Sentry Wizard:

<Include name="sourcemaps-wizard-instructions.mdx" />

If you want to configure source maps to upload manually, follow the guide for your bundler or build tool below.

## Sentry Bundler Support

- <PlatformLink to="/sourcemaps/uploading/webpack/">webpack</PlatformLink>
- <PlatformLink to="/sourcemaps/uploading/rollup/">Rollup</PlatformLink>
- <PlatformLink to="/sourcemaps/uploading/vite/">Vite</PlatformLink>
- <PlatformLink to="/sourcemaps/uploading/esbuild/">esbuild</PlatformLink>

### Cloudflare-Specific Tools

- <PlatformLink to="/sourcemaps/uploading/wrangler/">Wrangler</PlatformLink>

### Guides for Source Maps

- <PlatformLink to="/sourcemaps/uploading/typescript/">
    TypeScript (tsc)
  </PlatformLink>

  <Alert>

>     If you're using one of webpack, Vite, Rollup, or Esbuild, use the

Reviewer: ```suggestion
    If you're using a bundler like Webpack, Vite, Rollup, or Esbuild, use the
```

>     corresponding Sentry plugin instead (see section "Sentry Bundler Support").

Reviewer: ```suggestion
    corresponding Sentry plugin instead. For details, see the "Sentry Bundler Support" section.
```
